pinot-core/src/main/java/org/apache/pinot/core/auth/Actions.java

```diff
@@ -66,6 +67,7 @@ public static class Cluster {
     public static final String GET_TABLE_CONFIG = "GetTableConfig";
     public static final String GET_TABLE_LEADER = "GetTableLeader";
     public static final String GET_TASK = "GetTask";
+    public static final String GET_TASKTYPE_COUNT = "GetTaskTypeCount";
```

Comment:
We are actually getting the count of tasks instead of task types
```suggestion
    public static final String GET_TASK_COUNT = "GetTaskCount";
```